### DISCUSSION TITLE
Quacker Continuation

### DISCUSSION THREAD

**Discussion by android-smiley** [j-fbriere/squawker#1] [General]
Hi!

Thank you very much for continuing the work previously done in the Quacker project. Is there any information available on why that project disappeared?

Cheers!

**Comment by j-fbriere** [comment]
There was an explanation in the Quacker project's discussions but the owner has just deleted the project from github...
In a nutshell the reason is because of the recent changes made by Twitter which makes the app broken.

But I will wait to see what happens with Twitter before throwing the towel.

**Reply by android-smiley** [reply]
Thanks for the explanation! The frustration is understandable, but sad to see this project vanish. I'm optimistic that there'll be another workaround this time, so I'm glad and grateful, that you won't give in.

In case there won't be a good solution this time, could you consider just archiving this repo, in case Twitter reverses these policies some day?

**Reply by j-fbriere** [reply]
I'm also optimistic that either Twitter will reverse its changes or that some workaround will be found.

Don't worry about this repo, it will not disappear. It will remain active.

**Reply by ghost** [reply]
I shut down the repo due to the loss of passion and the constant breakages that have been made to Quacker.

**Reply by android-smiley** [reply]
That's understandable, the frustration on the development side must be even greater. Let's hope Twitter will turn around soon. Thank you very much for your reply and the great work on Quacker!

**Comment by ghost** [comment]
I have restarted development on Quacker. I have also fixed the feeds and tweet search again check it out here!
https://github.com/TheHCJ/Quacker

**Reply by tadcrazio** [reply]
Waiting anxiously for the release. Thanks for all the work you do.

**Reply by ghost** [reply]
Now built just remember **To install v3.0.0 you need to uninstall your existing version of Quacker first, as they aren't compatible with each other**

**Reply by tadcrazio** [reply]
Thanks man, installed and working!  I think every tweet on the feed is duplicated but will double check everything on my end before submitting an issue.

**Reply by ghost** [reply]
Seems I am also getting that issue, I'll work on a fix for it ASAP

**Comment by harishbewe** [comment]
What's the difference between quacker and squawker and any reason for the existence of two similar once? I'm getting confuse. Can anyone explain.

**Reply by j-fbriere** [reply]
I created Squawker first as a fork of Quacker because I wanted to help maintain Quacker, which as you probably know is a fork of Fritter with in addition the feeds functionality that Fritter abandoned.
I also help maintain Fritter for the functionalities that are in common with both projects.
But then the Quacker project was deleted by its owner. 
For any fork of Quacker to go on with the project after that, the owner asked to change the name of the fork, which I did.
Like I wrote in a previous answer, I will continue to maintain the Squawker project, especially that now most of the search/profile/subscription/feed functionalities for a guest access has been restored by Twitter.
I don't know what will happen to Quacker, which has been revived a couple of days ago. You'll have to ask the owner of the project.

**Reply by harishbewe** [reply]
Thanks for your response brother. I now understood 👍

**Reply by ghost** [reply]
Quacker is more experimental than squawker.

**Reply by ghost** [reply]
I am redirecting focus on Quacker to be a more streamlined and cutting edge experience
P.S I have restarted the project and will not be deleting the repo again as I have locked it

**Comment by ghost** [comment]
# Quacker has been discontinued since August 2023 and will recieve no further updates or support

**Comment by ghost** [comment]
@android-smiley @tadcrazio @j-fbriere @harishbewe @mrhcjones 
Read more: https://github.com/mrhcjones/Quacker/discussions/1
